staging/src/k8s.io/apiserver/pkg/server/httplog/httplog_test.go

```diff
@@ -111,7 +112,12 @@ func TestLogOf(t *testing.T) {
 					t.Errorf("Expected %v, got %v", test.want, got)
 				}
 			})
-			handler = withLogging(handler, DefaultStacktracePred, func() bool { return test.shouldLogRequest })
+			handler = withLogging(handler, DefaultStacktracePred, func() klog.Verbose {
+				if test.shouldLogRequest {
+					return klog.V(0)
+				}
+				return klog.V(1000)
```

Comment:
nit: using 1000 doesn't look good to me, I understand why you've modified `shouldLogRequest` but can we find a better way to do this?

Reply:
I can't think of a simpler one.